package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "opentoken-lib",
+    "version": "0.0.1",
```

Comment:
Yea, I think Kyle did the package but I don't really care either way. I am slightly disposed to starting from `1.0.0`. Just makes more sense. <major>.<minor>.<revision/or something>